webrender/src/frame_builder.rs

```diff
@@ -458,6 +458,7 @@ impl FrameBuilder {
             &viewport_rect,
             content_size,
             ScrollSensitivity::ScriptAndInputEvents,
+            false,
```

Comment:
Previously I set this to true, but it caused reftest failures, I'm wondering if its because of some special handling of the root scroll frame.

Reply:
Yes. Root scroll frame scrollbars are handled automatically by the ENABLE_SCROLLBARS option. We don't want this turned on in Gecko anyhow.